### PR TITLE
Added first task of Input/Output 

### PR DESCRIPTION
**Pull request by martinmilev**
Added first task of Input/Output that reads single line from console and prints result on the screen.

```java
 public static void main(String[] args) {
    String line = new ConsoleReader().readLine();
    System.out.println("Line: " + line);
  }
```

### CODE REVIEW

File: src/com/clouway/io/task1/ConsoleReader.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.clouway.io.task1;
+
+import java.util.Scanner;
+
+/**
+ * Created by clouway on 26.04.16.
+ */
+public class ConsoleReader {
+  public String readLine() {
```
Comment (ivanP91) [inline src/com/clouway/io/task1/ConsoleReader.java]: Missing comment.

File: src/com/clouway/io/task1/Demo.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.clouway.io.task1;
+
+/**
+ * Created by clouway on 26.04.16.
+ */
+public class Demo {
+  public static void main(String[] args) {
+    String line = new ConsoleReader().readLine();
```
Comment (ivanP91) [inline src/com/clouway/io/task1/Demo.java]: Please read at least 2 lines:
For example:
```java
ConsoleReader reader = new ConsoleReader();
String firstLine = reader.readLine();
String secondLine = reader.readLine();
System.out.println("First Line: " + firstLine);
System.out.println("Second line: " + secondLine);

```

File: src/com/clouway/io/task1/ConsoleReader.java (hunk 1 above)
Comment (martinmilev) [inline src/com/clouway/io/task1/ConsoleReader.java]: Done.

File: src/com/clouway/io/task1/Demo.java (hunk 2 above)
Comment (martinmilev) [inline src/com/clouway/io/task1/Demo.java]: Done.

**Comment by martinmilev** [comment]
All fixes are applied.

**Comment by ivanP91** [comment]
LGTM !

**Pull request merged**